nomos-services/key-management-system/src/backend/preload.rs

```diff
@@ -1,353 +1,122 @@
-//! This module contains a simple test implementation of [`KMSBackend`] that
```

Comment:
Most of the structures in this preload impl were mocks to be used only in tests. I've replaced the mocks with the proper structs so that the preload can actually be used outside of tests.